e2e_example/tool/build.dart

```diff
@@ -13,10 +13,10 @@ Future main(List<String> args) async {
   var builders = [
     apply('e2e_example', 'throwing_builder', [(_) => new ThrowingBuilder()],
         toRoot(),
-        hideOutput: true),
+        hideOutput: true, defaultGenerateFor: const ['test/**']),
```

Comment:
I don't think this needs to be added?

Reply:
looks like the cause of one of the travis failures actually

Reply:
ugh, I meant to add this on the TestBootstrapBuilder.